pkg/modules/application_mobility.go

```diff
@@ -399,10 +392,11 @@ func ApplicationMobilityPrecheck(ctx context.Context, op utils.OperatorConfig, a
 	}
 
 	// Check for secrets
-	appMobilitySecrets := []string{"license"}
+	ns := "default"
+	appMobilitySecrets := []string{"dls-license", "iv"}
 	for _, name := range appMobilitySecrets {
 		found := &corev1.Secret{}
-		err := r.GetClient().Get(ctx, types.NamespacedName{Name: name, Namespace: cr.GetNamespace()}, found)
+		err := r.GetClient().Get(ctx, types.NamespacedName{Name: name, Namespace: ns}, found)
```

Comment:
do we know DL license is always in default NS?
Could be a question for @harshitap26

Reply:
same question about license name

Reply:
@JacobGros Yes, right now the licensing service is creating the dls-license and iv secret in the default namespace.